app/src/main/java/com/sun/noteapp/ui/home/dialog/ViewDialog.kt

```diff
@@ -0,0 +1,26 @@
+package com.sun.noteapp.ui.home.dialog
+
+import android.content.Context
+import com.sun.noteapp.ui.base.BaseDialog
+import kotlinx.android.synthetic.main.dialog_view.*
+
+class ViewDialog(
+    context: Context,
+    layout: Int,
+    private val onUpdateViewTypeDialogCallback: OnLoadDialogCallback<Int>
+) : BaseDialog(context, layout){
+    override fun initListener() {
+        buttonSetViewList.setOnClickListener {
+            onUpdateViewTypeDialogCallback.Success(1)
```

Comment:
gán hằng số ứng với 1,2,3 để rõ 1 2 3 là cái gì 
```suggestion
            onUpdateViewTypeDialogCallback.Success(SomeThingTypes.LIST)
```